connections/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.134.0/containers/python-3/.devcontainer/base.Dockerfile
```

Comment:
This file has been updated since the 1.0.0 aca-py release. The easiest way to update it is probably just to copy the plugin_globals one.